src/Sil/Component/User/Model/Role.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licenced under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Sil\Component\User\Model;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @author Romain Sanchez <romain.sanchez@libre-informatique.fr>
+ */
+class Role implements RoleInterface
+{
+    /**
+     * name.
+     *
+     * @var string
+     */
+    protected $role;
```

Comment:
Should be `name` insteadof `role`. Usage will be `Role::getRole()`, it would be better used with `Role::getName()` ?

Reply:
property name is role to comply with symfony/security role